sdk/php/tests/ReferencesData/StaticReferenceTest.php

```diff
@@ -71,7 +71,7 @@ public function testGetFilePath(): void
      */
     public function testGetContent(): void
```

Comment:
Not important but `public function testGetData(): void` may be?

Reply:
done